scene/gui/color_mode.h

```diff
@@ -43,6 +43,7 @@ class ColorMode {
 
 	virtual int get_slider_count() const { return 3; };
 	virtual float get_slider_step() const = 0;
+	virtual float get_spinbox_arrow_step() const { return get_slider_step(); };
```

Comment:
Could also be `const = 0;`

Reply:
I was thinking it should have a default value so that individual color modes aren't required to implement this method.

Reply:
0 is the default value for SpinBox.custom_arrow_step, it will be ignored. From [docs](https://docs.godotengine.org/en/stable/classes/class_spinbox.html#class-spinbox-property-custom-arrow-step):
> `float custom_arrow_step = 0.0`
> If not 0, value will always be rounded to a multiple of custom_arrow_step when interacting with the arrow buttons of the SpinBox.